### ISSUE TITLE
Improve logging or metrics to make it easier for an operator to understand the current validator state

### ISSUE THREAD

**Issue by mvines** [solana-labs/solana#4952]
Reported from Discord
> I found it hard to follow if the node is running correctly and if has been a leader before
for example from the log, I see new fork:3604 parent:3603 but block height on the http://testnet.solana.com/ is 7036. Is it correct?

Perhaps every slot the validator can emit an easy to read log message that indicate its current state. 

cc: #4938

**Comment by aeyakovenko** [comment]
Idea: Leaderboard could display all the nodes and their current bank and slot heights.
